examples/example-vite-react-sdk/src/starknet-provider.tsx

```
@@ -0,0 +1,33 @@
+import { dojoConfig } from "../dojoConfig";
+import {
+    predeployedAccounts,
+    PredeployedAccountsConnector,
+} from "@dojoengine/predeployed-connector";
+
+let pa: PredeployedAccountsConnector[] = [];
+predeployedAccounts({
+    rpc: dojoConfig.rpcUrl as string,
+    id: "katana",
+    name: "Katana",
+}).then((p) => (pa = p));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)